smart_contracts.rst

```diff
@@ -760,6 +760,13 @@ Also, the amount that a participant can receive cannot be bigger than the total
 
 ``(6 R)`` is deduced from ``(5 R)``. It is needed by the Raiden client in order to not allow a participant to :ref:`withdraw <withdraw-channel>` more tokens from the on-chain channel deposit than he is entitled to.
 
+::
+
+    (7 R) T2 + L2 - T1 - L1 <= D2
+
+``T2 + L2 - T1 - L1`` is the netted total transferred amount from ``P2`` to ``P1``. This amount cannot be bigger than ``P2``'s deposit -> a participant cannot transfer more tokens than what he has in the channel, during the lifecycle of a channel.
```

Comment:
What about mentioning the lower bound? This value can be negative but not smaller than D1. (This is a corollary that must be enforce by P2)

Reply:
Correct, added.